README.md

```diff
@@ -26,7 +26,7 @@ Kata Containers combines technology from [Intel® Clear Containers](https://gith
 
 # Community
 
-Kata Containers is working to build a global, diverse and collaborative community. Anyone who is interested in supporting the technology is welcome to participate. We are seeking different expertise and skills, ranging from development, operations, documentation, marketing, community organization and product management.
+Kata Containers is working to build a global, diverse and collaborative community. Anyone who is interested in supporting the technology is welcome to participate. Learn how to contribute on the [Community pages](https://katacontainers.io/community/). We are seeking different expertise and skills, ranging from development, operations, documentation, marketing, community organization and product management.
```

Comment:
Thanks for raising this @mvincerx. I'm not sure if this link is the correct one though. Or, maybe https://katacontainers.io/community/ needs to be reworked: One that page under "How to Contribute", the first bullet is "
"Installation Guides", whereas I don't see a link to our main contributing guide:

- https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md

/cc @jimmytipit.

Reply:
I think we should land this now - we can discuss URL tweaks on follow-up PRs...